lib/modules/manager/cargo/extract.spec.ts

```diff
@@ -6,6 +6,7 @@ import { GlobalConfig } from '../../../config/global';
 import type { RepoGlobalConfig } from '../../../config/types';
 import { writeLocalFile } from '../../../util/fs';
 import type { ExtractConfig } from '../types';
+
```

Comment:
```suggestion
```

Reply:
Sorry, this was a miscommit. Actually it is a mitigation of the following issue:

For the following VSCode config:

```json
 "editor.codeActionsOnSave": {
    "source.organizeImports": true
  },
```

The `... from '.'` line below will be reordered on save and cause a hard error.